### PR TITLE
Added sample usage, corrected some typos

### PR DESCRIPTION
**Pull request by clamoris**

### CODE REVIEW

**Comment by kswaldemar** [comment]
Wow, nice. Someone finally correct my *not good enough* english grammar :)

File: README.md [hunk 1]
```diff
@@ -47,18 +47,24 @@ cmake --build . --config Release
...
+Also, see [client examples for official local runner](https://github.com/JustAMan/russian-ai-cup-visual/tree/master/clients).
 
-Then run viewer before strategy and it should start draw frames
+Sample usage: 
+
+1. Start the viewer.
+2. Start localrunner, preferably in render_to_screen=false mode.
+3. Start your strategy of choice.
+4. To be able to drew things in the viewer you will need to create a client, send data to the client in your strategy, and **close the frame**. 
+5. There is no need to close the viewer after the strategy is done, simply start from step 2.
```
Comment (kswaldemar) [inline README.md]: There is no need to close the viewer after the strategy is done, simply clean old data (Edit->Clear frame data, or just Ctrl+R) and start from step 2.

File: README.md [hunk 2]
```diff
@@ -47,18 +47,24 @@ cmake --build . --config Release
...
-Also see [client examples for official local runner](https://github.com/JustAMan/russian-ai-cup-visual/tree/master/clients).
+Also, see [client examples for official local runner](https://github.com/JustAMan/russian-ai-cup-visual/tree/master/clients).
 
-Then run viewer before strategy and it should start draw frames
+Sample usage: 
+
+1. Start the viewer.
+2. Start localrunner, preferably in render_to_screen=false mode.
+3. Start your strategy of choice.
+4. To be able to drew things in the viewer you will need to create a client, send data to the client in your strategy, and **close the frame**.
```
Comment (kswaldemar) [inline README.md]: I think `**close the frame** with special command`. Would be better

**Comment by clamoris** [comment]
Done.

**Comment by kswaldemar** [comment]
Great :+1:

**Pull request merged**